ads/custom.md

```diff
@@ -0,0 +1,186 @@
+      </a>
+    </template>
+</amp-ad>
+<!-- The ad server will be called with the URL https://mysite/my-ad-server -->
+```
+
+### Two ads with different slots
+
+```html
+<amp-ad width=300 height=250
+    type="custom"
+    data-slot="1"
+    data-url="https://mysite/my-ad-server?someparam=somevalue"
+>
+    <template type="amp-mustache" id="amp-template-id">
+      <a href="{{href}}">
+        <amp-img layout='fixed' height="300" width="250" src="{{src}}" data-info="{{info}}"></amp-img>
+      </a>
+    </template>
+</amp-ad>
+<amp-ad width=400 height=300
+    type="custom"
+    data-slot="2"
+    data-url="https://mysite/my-ad-server?someparam=somevalue"
+>
+    <template type="amp-mustache" id="amp-template-id">
+      <a href="{{href}}">
+        <amp-img layout='fixed' height="400" width="300" src="{{src}}" data-info="{{info}}"></amp-img>
+      </a>
+    </template>
+</amp-ad>
+<!-- The ad server will be called with the URL https://mysite/my-ad-server?someparam=somevalue&ampslots=1,2 -->
+```
+
+### Ads from different ad servers
+```html
+<amp-ad width=300 height=250
+    type="custom"
+    data-slot="slot-name-a"
+    data-url="https://mysite/my-ad-server"
+>
+    <template type="amp-mustache" id="amp-template-id">
+      <a href="{{href}}">
+        <amp-img layout='fixed' height="300" width="250" src="{{src}}" data-info="{{info}}"></amp-img>
+      </a>
+    </template>
+</amp-ad>
+<amp-ad width=400 height=300
+    type="custom"
+    data-slot="slot-name-b"
+    data-url="https://mysite/my-ad-server"
+>
+    <template type="amp-mustache" id="amp-template-id">
+      <a href="{{href}}">
+        <amp-img layout='fixed' height="400" width="300" src="{{src}}" data-info="{{info}}"></amp-img>
+      </a>
+    </template>
+</amp-ad>
+<amp-ad width=300 height=250
+    type="custom"
+    data-slot="123"
+    data-url="https://my-other-site/my-other-ad-server"
+>
+    <template type="amp-mustache" id="amp-template-id">
+      <a href="{{href}}">
+        <amp-img layout='fixed' height="300" width="250" src="{{src}}" data-info="{{info}}"></amp-img>
+      </a>
+    </template>
+</amp-ad>
+<!-- Two ad server calls will be made: -->
+<!-- The first:  https://mysite/my-ad-server?ampslots=slot-name-a,slot-name-b -->
+<!-- The second: https://my-other-site/my-other-ad-server?ampslots=123 -->
+```
+
+## Supported parameters
+
+### data-url (mandatory)
+
+This must be starting with `https://`, and it must be the address of an ad
+server returning json in the format defined below.
+
+### data-slot (optional)
+
+On the assumption that most pages have multiple ad slots, this is passed to the
+ad server to tell it which slot is being fetched. This can be any alphanumeric string.
+
+If you have only a single ad for a given value of `data-url`, it's OK not to bother with
+the slot id. However, do not use two ads for the same `data-url` where one has a slot id
```

Comment:
It just makes it a lot easier to read the source in Netbeans... And for some reason, my Markdown Plus previewer in Chrome isn't working.